src/components/Observations/ObsPreviewImage.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import classnames from "classnames";
+import { ImageBackground, View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import LinearGradient from "react-native-linear-gradient";
+
+type URI = {
+  uri: string
+}
+
+type Props = {
+  uri?: URI,
+  opaque?: boolean
+};
+
+const ObsPreviewImage = ( { uri, opaque = false }: Props ): Node => ( uri ? (
+  <ImageBackground
+    source={uri}
+    className={classnames( "grow aspect-square", { "opacity-50": opaque } )}
+    testID="ObsList.photo"
+  >
+    <LinearGradient
```

Comment:
For the ObsListItem we do not have this gradient overlay.

Reply:
I see, originally i did not have it but the icon was difficult to see on some images. i assumed it would follow the grid layout. i can change this